dts/bindings/adc/nxp,gau-adc.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  NXP GAU GPADC.
+
+compatible: "nxp,gau-adc"
+
+include:
+  - name: base.yaml
+  - name: adc-controller.yaml
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  nxp,clock-divider:
+    type: int
+    description: |
+      Clock divider from 1 to 32.
+      Default is 1 which is reset value.
+    default: 1
+
+  nxp,power-mode:
+    type: string
+    description: |
+      Current bias.
+      Default is "full-bias" because it is the reset value.
+    enum:
+      - "full-bias"
+      - "half-bias"
+    default: "full-bias"
+
+  nxp,trigger-source:
+    type: string
+    description: |
+      Conversion trigger source.
+      Default is "software" which is the only value currently supported".
+    enum:
+      - "gpt"
+      - "acomp"
+      - "gpio40"
```

Comment:
These values look like they will be specific to the RW61x SOC. I guess we only have that reference manual to go on though for these names. Any thoughts here on what we could call these that would be more generic?

Perhaps we should simply direct users to the reference manual for the meanings of each value?

Reply:
Yeah I'm actually just going to delete this property